src/index.html

```diff
@@ -1,13 +1,378 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Crazybaby</title>
+    <link
+      rel="icon"
+      href="/images/favicon.png"
+      type="image/x-icon">
```

Comment:
```suggestion
    <link
      rel="icon"
      href="/images/favicon.png"
      type="image/x-icon"
    >
```

Reply:
![image](https://user-images.githubusercontent.com/53319174/182219193-da7e86b1-6cf6-48a4-a660-fe4f209b2bca.png)

This is the color in Figma. But I agree. Probably it is really better to replace it

Reply:
You should reset fields on submit )